tools/fetch-team-data.js

```diff
@@ -96,7 +96,7 @@ async function fetchUserProfile(username) {
 			: profile.blog
 				? `https://${profile.blog}`
 				: profile.blog,
-		avatar_url: profile.avatar_url,
+		avatar_url: `${profile.avatar_url.replace("private-", "").split("?")[0]}?v=4`,
```

Comment:
Some URLs have `u` parameter:

https://github.com/eslint/eslint.org/blob/617c1962e2b99163cf69f6ad217eea5e7d9a441c/src/_data/sponsors.json#L76-L77

Any ideas about what it represents?

Should we maybe remove just the `jwt` parameter and leave the others?

Reply:
>Some URLs have u parameter:

Can it stands for unique? even though without this parameter the results are same.

Also seems links are already fixed by commit https://github.com/eslint/eslint.org/commit/e385d5fc1a47df1d491509c4aabf754459bb5d42 should we still apply this fix?

Reply:
> Also seems links are already fixed by commit [e385d5f](https://github.com/eslint/eslint.org/commit/e385d5fc1a47df1d491509c4aabf754459bb5d42) should we still apply this fix?

I also noticed that it might be a good idea to wait a couple of days. If GitHub is still testing features with a subset of users, it could potentially break again. As of now, we might not be part of the experiment, but if they do a full rollout, we could be impacted.

Reply:
I think it would be good to continue working on this PR and have it ready (or perhaps even merge it) in case the problem happens again. Unless they've decided not to continue with the change, but I couldn't find any mention of that in https://github.com/orgs/community/discussions/147297.

Reply:
>Should we maybe remove just the jwt parameter and leave the others?

should we work on this approach? as we can see both links with and without u token gives the same result:
[`https://avatars.githubusercontent.com/u/53535748?u=d1e55d7661d724bf2281c1bfd33cb8f99fe2465f&v=4`](https://avatars.githubusercontent.com/u/53535748?u=d1e55d7661d724bf2281c1bfd33cb8f99fe2465f&v=4)

[`https://avatars.githubusercontent.com/u/53535748?v=4`](https://avatars.githubusercontent.com/u/53535748?v=4)

Reply:
I think it's better to remove the JWT parameter and retain the others.